src/menu/views/cpakfs_manager.c

```
@@ -620,21 +646,21 @@ static void draw (menu_t *menu, surface_t *d) {
     if (has_mem) {
-        sprintf(has_mem_text, "Controller Pak detected");
+            snprintf(has_mem_text, sizeof(has_mem_text), "Controller Pak detected");
         style = STL_GREEN;
 
         if (has_mem && !corrupted_pak) {
             style = STL_GREEN;
-            sprintf(free_space_cpak_text, "%d/123 free blocks", cpakfs_stats.pages.total - cpakfs_stats.pages.used);
+            snprintf(free_space_cpak_text, sizeof(free_space_cpak_text), "%d/123 free blocks", cpakfs_stats.pages.total - cpakfs_stats.pages.used);
         } else if (has_mem && corrupted_pak) {
-            sprintf(has_mem_text, "%s %s", has_mem_text, " (is NOT valid. Corrupted)");
+            snprintf(has_mem_text, sizeof(has_mem_text), "%s %s", has_mem_text, " (is NOT valid. Corrupted)");
```

Cppcheck (2.20.0)

[error] Undefined behavior

(sprintfOverlappingData)